### ISSUE TITLE
Regarding Validation Code for PASCAL VOC

### ISSUE THREAD

**Issue by indersingh17188** [Alibaba-MIIL/ASL#62]
Hello again,

Thank you so much for your quick feedback on my last issue. Appreciate it.

I would like to know if you there is any validation code for PASCAL-VOC dataset in order to reproduce the same results.

Thanks,
Inder

**Comment by mrT23** [comment]
i released the model of pascal.

regarding validation code, it is very similar to MS-COCO code
https://github.com/Alibaba-MIIL/ASL/blob/main/validate.py
just load the PASCAL dataset instead

**Comment by indersingh17188** [comment]
Ok, thank you so much for your quick response.

**Comment by indersingh17188** [comment]
Hello again,

I tried to load the PASCAL VOC dataset and train TResNet-XL using ImageNet Pretrained weights but after 3 or 4 epochs, the Loss started to go negative by a huge number and max accuracy achieved was 82% (using the same mAP function as for MS-COCO). Could you please, from a high level perspective, share your thoughts on what is going wrong here?

Thanks in advance.

Regards,
Inder
![image](https://user-images.githubusercontent.com/44872355/125448082-cfae0c7d-60e1-4b2d-b042-1faef1a3f6b2.png)
![image](https://user-images.githubusercontent.com/44872355/125448096-9b5cec85-4808-4ebc-a0b9-3dbe941a1b15.png)

**Comment by mrT23** [comment]
to be honest, i don't know. but you can try several things (do all of them)
- move to tresnet_m. dont start training with a large model. make everything work first on  a smaller model
- move to 21K pretraining https://github.com/Alibaba-MIIL/ImageNet-21K-P
- use adam with a learning rate of 5e-5 instead 3e-4
- make sure your code has proper warmup, and that you are using EMA

**Comment by indersingh17188** [comment]
Hello again,

There was a problem with the dataloader itself. Now it is resolved hence closing the issue.

Thanks for your quick feedback.

Regards,
Inder
